fuse/ipns/ipns_unix.go

```diff
@@ -370,13 +384,21 @@ func (dir *Directory) Mkdir(ctx context.Context, req *fuse.MkdirRequest) (fs.Nod
 
 func (fi *File) Open(ctx context.Context, req *fuse.OpenRequest, resp *fuse.OpenResponse) (fs.Handle, error) {
 	if req.Flags&fuse.OpenTruncate != 0 {
-		log.Warning("Need to truncate file!")
+		log.Info("Need to truncate file!")
 		err := fi.fi.Truncate(0)
 		if err != nil {
 			return nil, err
 		}
 	} else if req.Flags&fuse.OpenAppend != 0 {
-		log.Warning("Need to append to file!")
+		log.Info("Need to append to file!")
+
+		// seek(0) essentially resets the file object, this is required for appends to work
+		// properly
+		_, err := fi.fi.Seek(0, os.SEEK_SET)
```

Comment:
fuse kinda handles this for us, and now that I think about it, this seek(0) should really just go at the top of the function.

All reads at 'ReadAt' and all writes are 'WriteAt', so you dont have to worry about what offset your file descriptor is at. The seek(0) resets the cache buffer for the ipnsfs file, I should probably just have a `Reset` method on it instead of doing this.